samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/Login.java

```diff
@@ -60,7 +60,8 @@ public void clicks_login_button() {
 	@Then("^she is redirected to the Root View$")
 	public void redirected_to_root_view() {
 		rootPage.waitForWebElementDisplayed(rootPage.profileTable);
-		Assert.assertTrue("Can't access profile information", rootPage.profileTable.isDisplayed());
```

Comment:
You don't need to change this. This error message is displayed when assertion fails. So the message shouldn't change.

Reply:
Got it, will revert the change now. Thanks